### DISCUSSION TITLE
騙子症候群

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1557] [General]
冒名頂替者症候群，或稱騙子症候群
指的是人在獲得一定成就後
會懷疑這只不過是運氣好而非我實力屌
自己其實是個不配得到大家肯定的 #騙子
_
這種現象不只出現在成功人士身上
美國研究就發現，有70%的普通人
也出現了這種懷疑自己能力的心理狀態
_
這在跨入競爭力更高的新環境時尤其常見：
你很容易發現周遭的同學 / 同事表現都超罩
而自己卻像走錯教室 / 辦公室般在旁邊瞎鬧
_
雖然這種自我懷疑，可能會讓你激勵自己
要更努力工作努力學習、避免被識破被排擠
但同時，那種再也容不下錯誤的焦慮及不安
會讓你更不敢做出冒險性的嘗試
最終將你的信心推向崩潰邊緣
因此有一份能坦然接受外界肯定的自信
也是一個心態上的重要課題

**Comment by poweichen00** [comment]
接受外界肯定並培養一份坦然的自信是非常重要的心態課題，要意識到自己的成就是有價值的，並相信自己能達到這些成就所需的能力，理解每個人都會犯錯和遇到挑戰。

**Comment by larry40305** [comment]
雖然這種自我懷疑可能激勵你更加努力工作和學習，以避免被揭露或排擠，但同時也可能帶來焦慮和不安感。這種焦慮可能使你不敢冒險嘗試新的事物，並最終破壞你的自信心。因此，擁有能夠坦然接受外界肯定的自信心是一個重要的心態課題。

**Comment by ThanatosJun** [comment]
自信心的建立在一定的時間與成功上，自信心讓人能夠勇於面對未來的挑戰。對一位棒球打擊者來說，當他站上打擊區，自信心多能讓打者正視投手，較能擊出較佳打位。如果沒有自信心的人，我這邊有一個運動員常使用的方式，找出自己最舒適的姿勢，在感到焦躁不安時重複該動作可以減輕焦慮，重返佳態。

**Comment by Alex11392** [comment]
我覺得自信和對自己的認可是基於自己的努力和堅持，而非外界或社會所定義的成功，也許從這樣的角度思考就能避免自己陷入騙子症候群。
